lms/tasks/mailchimp.py

```diff
@@ -4,6 +4,26 @@
 from lms.tasks.celery import app
 
 
+@app.task(
+    acks_late=True,
+    autoretry_for=(Exception,),
+    max_retries=2,
+    retry_backoff=3600,
+    retry_backoff_max=7200,
+)
+def send(*, sender, recipient, **kwargs) -> None:
+    """Send an email using Mailchimp's API."""
+
+    sender = EmailSender(**sender)
+    recipient = EmailRecipient(**recipient)
+
+    with app.request_context() as request:  # pylint:disable=no-member
+        mailchimp_service = request.find_service(name="mailchimp")
+        print(request.environ)
+        with request.tm:
+            mailchimp_service.send(sender=sender, recipient=recipient, **kwargs)
```

Comment:
This is exactly the same as the `send_template()` Celery task below except it calls `send()` instead of `send_template()`